rct_optimizations/CMakeLists.txt

```diff
@@ -65,11 +67,18 @@ if(CATKIN_ENABLE_TESTING)
   catkin_add_gtest(${PROJECT_NAME}_conversion_tests test/conversion_utest.cpp)
   target_link_libraries(${PROJECT_NAME}_conversion_tests ${PROJECT_NAME}_test_support)
 
+  catkin_add_gtest(${PROJECT_NAME}_ceres_math_utilities_tests test/ceres_math_utilities_utest.cpp)
+  target_link_libraries(${PROJECT_NAME}_ceres_math_utilities_tests ${PROJECT_NAME}_test_support)
+
   catkin_add_gtest(${PROJECT_NAME}_extrinsic_multi_static_camera_tests test/extrinsic_multi_static_camera_utest.cpp)
   target_link_libraries(${PROJECT_NAME}_extrinsic_multi_static_camera_tests ${PROJECT_NAME}_test_support)
 
   catkin_add_gtest(${PROJECT_NAME}_extrinsic_camera_on_wrist_tests test/extrinsic_camera_on_wrist_utest.cpp)
   target_link_libraries(${PROJECT_NAME}_extrinsic_camera_on_wrist_tests ${PROJECT_NAME}_test_support)
+
+  catkin_add_gtest(${PROJECT_NAME}_extrinsic_static_camera_robot_calibration_tests test/extrinsic_static_3d_camera_robot_calibration_utest.cpp)
+  target_link_libraries(${PROJECT_NAME}_extrinsic_static_camera_robot_calibration_tests ${PROJECT_NAME}_test_support)
+
```

Comment:
CI is failing because one or both of these files does not yet exist

Reply:
removed